samples/snippets/csharp/VS_Snippets_Wpf/WindowClosingSnippets/CSharp/DataWindow.xaml.cs

```diff
@@ -1,4 +1,3 @@
-//<SnippetWindowClosingCODEBEHIND1>
 using System; // EventArgs
 using System.ComponentModel; // CancelEventArgs
 using System.Windows; // window
```

Comment:
Are comments indicating why a `using` is needed actually useful? (Though this applies to more than just this snippet, so maybe that should be a separate PR.)

Also, why is the namespace here `CSharp` (and `VisualBasic`), when all the other snippets use `SDKSample`?